core/kazoo_number_manager/src/knm_util.erl

```diff
@@ -127,9 +126,3 @@ read_fixture({'ok', Contents}, _F) ->
     kz_util:to_list(Contents);
 read_fixture({'error', 'enoent'}, F) ->
     throw({'error', 'missing_fixture', F}).
-
-%% TODO
-%% this should be replaced with a call to elibphonenumber
-%% when/if we integrate that lib or do it ourselves
--spec prefix_for_country(ne_binary()) -> ne_binary().
-prefix_for_country(<<"US">>) -> <<"+1">>.
```

Comment:
as commented above, leave this and change it to call  `knm_iso3166a2_itu:to_itu`